pokemongo_bot/cell_workers/move_to_fort_worker.py

```diff
@@ -41,6 +43,8 @@ def work(self):
                 while True:
                     if step_walker.step():
                         break
+                    else:
+						self.bot._work_on_cell_catch(self.position)
```

Comment:
What do you think if the _work_on_cell_catch logic was put away into a util of sorts?